apps/prairielearn/elements/pl-sketch/pl-sketch.py

```diff
@@ -0,0 +1,36 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+def prepare(element_html, data):
+    return data
+
+
+def render(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    width = pl.get_integer_attrib(element, "width", 600)
+    height = pl.get_integer_attrib(element, "height", 700)
+    if data["panel"] == "question":
```

Comment:
You can unindent this by returning immediately if the panel not `"question"`.

Reply:
good suggestion, thanks!